src/freenas/etc/ix.rc.d/ix-syslogd

```diff
@@ -12,7 +12,7 @@
 generate_syslog_conf()
 {
 	local IFS="|"
-	local f="stg_syslogserver stg_sysloglevel"
+	local f="adv_syslogserver adv_sysloglevel"
```

Comment:
You have changed the fields, but did not change the table to system_advanced below.